link/perf_event.go

```diff
@@ -67,10 +67,86 @@ const (
 	uretprobeEvent
 )
 
+// perfEventLink wraps perfEvent and RawLink.
+type perfEventLink struct {
+	pe  *perfEvent
+	raw *RawLink
```

Comment:
An unattached `perfEventLink` shouldn't exist, so this can just embed `RawLink` directly, see `NetNsLink` for example. Saves us a bunch of boilerplate.